winml/api/Microsoft.AI.MachineLearning.Experimental.idl

```diff
@@ -139,7 +145,7 @@ namespace ROOT_NS.AI.MachineLearning.Experimental {
     //! The Save method serializes the model as an ONNX model to a specified path.
     void Save(String file_name);
 
-    //! The JoinModel fuses two models by linking  outputs from the first model, to inupts of the second. 
+    //! The JoinModel fuses two models by linking  outputs from the first model, to inupts of the second.
```

Comment:
double spaces after `linking`, and `inupts` -> `inputs` (preexisting)

Reply:
removed double space